samples/hybrid-connections/java/simple-http-demo/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
Both [SB](https://github.com/Azure/azure-service-bus-java/blob/dev/.gitignore#L25) and [EH](https://github.com/Azure/azure-event-hubs-java/blob/dev/.gitignore#L75) have .classpath and .project in their .gitignore files. Should we do the same?

Reply:
If it helps the user run the sample, though, maybe it's good to keep them.

---
In reply to: [242641497](https://github.com/Azure/azure-relay/pull/35#discussion_r242641497) [](ancestors = 242641497)